forem/forem
# Add `Edit Profile Image` option to Onboarding

## What type of PR is this? (check all applicable)


## Description

Currently onboarding process does not have `Edit Profile Image` option. This PR introduces that option with current limtation.

## Related Tickets & Documents



- Closes #19497

## QA Instructions, Screenshots, Recordings

1. Visit `/onboarding` on localhost or Uffizzi. (http://localhost:3000/onboarding)
2. Follow the process and notice the `Edit Profile Image` option. Select image and update the profile image from here directly.

| Before | After |
| ------ | ------ |
| ![Before](https://github.com/forem/forem/assets/84626054/d448d26b-226d-4e65-bca3-a810c7a65349) | ![Image uploaded](https://github.com/forem/forem/assets/84626054/e9e4b758-9d24-4687-aaee-53935dc1df02) |

**Output on mobile devices:**

https://github.com/forem/forem/assets/9396084/5146b690-072f-4e6c-b795-2d8fc2434d58

https://github.com/forem/forem/assets/9396084/da347be4-1e82-47f5-812e-42ea68f68773


**Light mode and Dark Mode**
| Light Mode  | Dark Mode |
| ------------- | ------------- |
| <img width="700" alt="Screenshot 2023-06-14 at 12 36 56 PM" src="https://github.com/forem/forem/assets/9396084/d8749d31-1083-45d6-a4ec-88b8eccfdda2"> | <img width="697" alt="Screenshot 2023-06-14 at 12 36 39 PM" src="https://github.com/forem/forem/assets/9396084/f190ba60-5759-4918-8b1b-2f705825b672"> |
| <img width="701" alt="Screenshot 2023-06-14 at 12 37 07 PM" src="https://github.com/forem/forem/assets/9396084/29e3ef73-1433-45b1-91c8-e7653ab0d0cf"> | <img width="702" alt="Screenshot 2023-06-14 at 12 36 30 PM" src="https://github.com/forem/forem/assets/9396084/6f4bc009-d663-4607-a69d-168b24171ffa"> |



### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_
- [x] Semantic HTML implemented? Yes
- [x] Keyboard operability supported? Yes
- [x] Checked with [axe DevTools](https://www.deque.com/axe/) and addressed `Critical` and `Serious` issues? Yes, 0 issues.
- [x] Color contrast tested? Yes

<img width="300" alt="Screenshot 2023-06-14 at 9 24 45 PM" src="https://github.com/forem/forem/assets/9396084/977c8f12-cf48-4a2a-8bde-f18edbcb08a1">

_For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?
